BlueprintUICommonControls/Sources/EditingMenu.swift

```diff
@@ -0,0 +1,309 @@
+
+        var items: [EditingMenuItem] = []
+
+        private let longPress: UILongPressGestureRecognizer
+
+        override init(frame: CGRect) {
+            longPress = UILongPressGestureRecognizer()
+
+            super.init(frame: frame)
+
+            longPress.addTarget(self, action: #selector(handleLongPress))
+
+            addGestureRecognizer(longPress)
+        }
+
+        @available(*, unavailable)
+        required init?(coder: NSCoder) { fatalError() }
+
+        // MARK: UIView
+
+        override var canBecomeFirstResponder: Bool {
+            true
+        }
+
+        @objc override func canPerformAction(_ action: Selector, withSender sender: Any?) -> Bool {
+            items.map(\.kind.selector).contains(action)
+        }
+
+        // MARK: UIResponderStandardEditActions
+
+        @objc override func cut(_ any: Any?) {
+            perform(kind: .cut)
+        }
+
+        @objc override func copy(_ any: Any?) {
+            perform(kind: .copy)
+        }
+
+        @objc override func paste(_ any: Any?) {
+            perform(kind: .paste)
+        }
+
+        @objc override func delete(_ any: Any?) {
+            perform(kind: .delete)
+        }
+
+        @objc override func select(_ any: Any?) {
+            perform(kind: .select)
+        }
+
+        @objc override func selectAll(_ any: Any?) {
+            perform(kind: .selectAll)
+        }
+
+        @objc override func toggleBoldface(_ any: Any?) {
+            perform(kind: .toggleBoldface)
+        }
+
+        @objc override func toggleItalics(_ any: Any?) {
+            perform(kind: .toggleItalics)
+        }
+
+        @objc override func toggleUnderline(_ any: Any?) {
+            perform(kind: .toggleUnderline)
+        }
+
+        @objc override func increaseSize(_ any: Any?) {
+            perform(kind: .increaseSize)
+        }
+
+        @objc override func decreaseSize(_ any: Any?) {
+            perform(kind: .decreaseSize)
+        }
+
+        @available(iOS 15.0, *)
+        @objc override func printContent(_ any: Any?) {
+            perform(kind: .printContent)
+        }
+
+        @available(iOS 15.0, *)
+        @objc override func pasteAndGo(_ any: Any?) {
+            perform(kind: .pasteAndGo)
+        }
+
+        @available(iOS 15.0, *)
+        @objc override func pasteAndMatchStyle(_ any: Any?) {
+            perform(kind: .pasteAndMatchStyle)
+        }
+
+        @available(iOS 15.0, *)
+        @objc override func pasteAndSearch(_ any: Any?) {
+            perform(kind: .pasteAndSearch)
+        }
+
+        private func perform(kind: EditingMenuItem.Kind) {
+            let action = items.first {
+                $0.kind == kind
+            }
+
+            action?.onSelect()
+        }
+
+        @objc private func handleLongPress() {
+
+            guard longPress.state == .began else { return }
+
+            _ = showMenu()
+        }
+
+        private func showMenu() -> Bool {
```

Comment:
If this bool isn't used anywhere, does it make sense to keep it? (_Should_ we be using it somehow?)

Reply:
True – I was using this for an accessibility integration, but removed that for the time being since I need to figure out the correct way to do it (rotors).